BranchMetrics/cpp-branch-deep-linking-attribution
# [CORE-1675] MSI improvements

## Reference
CORE-1675
SDK-1116

## Description
This change introduces a signed MSI binary. It also adds all .pdb files from Poco & other dependencies to eliminate a lot of link-time warnings in debug builds and enable further debugging options.

Open-source license files for the Branch SDK and all third-party dependencies are included and installed under C:\Program Files (x86)\Branch SDK\licenses. Third-party licenses are installed with the third-party dependencies (i.e. not installed if not selected).

The draft setup docs and the old examples (other than TestBed variants) have been removed.

## Testing Instructions
It's not necessary to build the MSI. It's attached here as an asset: https://github.com/BranchMetrics/cpp-branch-deep-linking-attribution/releases/download/1.2.0-rc4/BranchSDK.msi. It should be possible to see the digital signature from Branch Metrics in the Properties tab in File Explorer once downloaded. It will install a number of .pdb files other than BranchIO.pdb in C:\Program Files (x86)\Branch SDK\lib\x86|x64\Debug. Building any of the TestBed variants against this installation will link cleanly without complaints about missing .pdb files.
